dev_guide/integrating_external_services.adoc

```diff
@@ -83,11 +83,12 @@ endpoints manually:
         name: "mysql"
         protocol: "TCP"
         port: 3306
-        targetPort: 3306
+        targetPort: 3306 <1> 
         nodePort: 0
-  selector: {} <1>
+  selector: {} <2>
 ----
-<1> The `selector` field to leave blank.
+<1> Optional: Port on the backing pods to which the service forwards connections.
```

Comment:
s/Port/The port  (x2)

I'd say "that the service forwards connections to." (Style says "Use a preposition at the end of a sentence to avoid an awkward or stilted construction." This isn't too bad, but moving the preposition does make it read a little easier.)